### DISCUSSION TITLE
[Support]: Frigate 0.14 iOS missing maximize button

### DISCUSSION THREAD

**Discussion by Produce2727** [blakeblackshear/frigate#12902] [General Support] [answered]
### Describe the problem you are having

Under Android or at PC in web browser you can maximize the video playback, but under iOS the button is missing.

### Steps to reproduce

Click the history button and play a video.

### Version

0.14

### Frigate config file

```yaml
...
```

### Relevant log output

```shell
...
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

...

**Comment by hawkeye217** [comment, marked as answer]
This is an iOS limitation. It likely will be fixed in the next iOS version.

**Reply by Produce2727** [reply]
Thank you! 😅 What's the specific name of the limitation? Why exactly does it not work?

**Reply by NickM-27** [reply]
see https://forums.developer.apple.com/forums/thread/133248

**Reply by NickM-27** [reply]
there is also the api documentation https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API#browser_compatibility
